<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormFournisseur</class>
 <widget class="QDialog" name="FormFournisseur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter Fournisseur</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>152</width>
     <height>196</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditRef">
      <property name="placeholderText">
       <string>Réf Fournisseur</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditRaison">
      <property name="placeholderText">
       <string>Raison Sociale</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditNIF">
      <property name="placeholderText">
       <string>NIF</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditAdresse">
      <property name="placeholderText">
       <string>Adresse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditTel">
      <property name="placeholderText">
       <string>Téléphone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAjouter">
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
